tests/test_create_sentence.rs

```diff
@@ -0,0 +1,8 @@
+extern crate reqwest;
+
+#[test]
+fn test_create_sentence_returns_200() {
+
+    let response = reqwest::get("http://localhost:8000/sentences").unwrap();
+    assert!(response.status().is_success());
```

Comment:
hmmm they named 200 , is_success ?  or is_success a more global check for all 2XX responses ?